test/base/dune

```diff
@@ -3,8 +3,10 @@
  (deps
   (:test test.ml)
   (glob_files %{project_root}/src/.ppxlib.objs/byte/*.cmi))
- (action (chdir %{project_root}
-          (progn
-           (ignore-outputs
-            (run %{project_root}/test/expect/expect_test.exe %{test}))
-           (diff? %{test} %{test}.corrected)))))
+ (action
+  (chdir %{project_root}
+   (progn
+    (setenv OCAMLRUNPARAM ""
```

Comment:
I personally prefer to include things like this in the code. I.e:

```
let () = Printexc.record_backtrace false
```

However, this is a matter of taste so you can have the final word here.

Reply:
That's what I tried at first but couldn't get it working for some reasons, I'll try again!

Reply:
I moved it to the test file although what I really wanted to do was to disable backtraces for all expect tests in `expect_test.mll` but I couldn't find a better solution than using `Toploop.execute_phrase` and that's not really satisfying.

Now that I think about it I'm not even sure we want to disable backtraces for all expect test so I'll stick to that for now!